mitodl/open-discussions
# Audio player

#### Pre-Flight checklist


#### What are the relevant tickets?
https://trello.com/c/C0KaEV0e/268-podcasts-build-a-simple-podcast-player

#### What's this PR do?
Adds an audio player component to open-discussions based on amplitudejs, mainly for podcasts but it could be used with any audio.

#### How should this be manually tested?
Run open-discussions and visit `/podcasts`.  Click play on any episode and it should open the player.  Try different resolutions, podcasts and run the tests.  Try adjusting the playback speed as well as seeking by buttons and the seek bar.

#### Screenshots (if appropriate)
![Screen Shot 2020-04-22 at 5 29 06 PM](https://user-images.githubusercontent.com/12089658/80035951-e6db3600-84be-11ea-8d8a-2d165e28bfc7.png)
![Screen Shot 2020-04-22 at 5 29 32 PM](https://user-images.githubusercontent.com/12089658/80035970-ecd11700-84be-11ea-8f92-2d2ac420ee6c.png)


#### What GIF best describes this PR or how it makes you feel?
![swanson](https://user-images.githubusercontent.com/12089658/80121465-55b99d00-855a-11ea-9901-248c72f5c449.gif)
